templates/default/environment_file.erb

```diff
@@ -0,0 +1,23 @@
+DAEMON_ARGS=" -d -c <%= @launch_options['conf_file'] %>
```

Comment:
Yes, I've experienced this also. Need to either remove `-d` or add `expect daemon` to the service definition.